config/templates/stack-template.json

```diff
@@ -89,14 +88,22 @@
       },
       "DependsOn" : ["AutoScaleController"]
     },
+    {{end}}
     "AutoScaleController": {
       "Type": "AWS::AutoScaling::AutoScalingGroup",
       "Properties": {
         "AvailabilityZones": [
```

Comment:
Good point. If nobody disagrees I will remove the availability zone properties from the template tomorrow.